annotation/src/main/kotlin/com/tobrun/datacompat/annotation/Default.kt

```diff
@@ -10,7 +10,13 @@ package com.tobrun.datacompat.annotation
  * @param valueAsString exact representation of the default value. E.g. if default [String] is used,
  *  it should be passed here as "\"STRING_VALUE\""; if default [Int] is used, it should be passed
  *  as "INT_VALUE".
+ *
+ * @param importList if default parameter requires additional imports, they should be passed here.
+ *  E.g. `listOf("android.graphics.Color")`
  */
 @Retention(AnnotationRetention.RUNTIME)
 @Target(AnnotationTarget.VALUE_PARAMETER)
-annotation class Default(val valueAsString: String)
+annotation class Default(
+    val valueAsString: String,
+    val importList: Array<String> = []
```

Comment:
just a question. why is it an array?

Reply:
lists are not supported as annotation class parameter - will rename the variable then; if you were asking why is it actually a collection - because there could possibly be several imports when defining default value

Reply:
I'm not sure I follow - we could have default value like `Color(Date(CustomClass("John")))` - then in my case it'll be:
```
@Default("Color(Date(CustomClass(\"John\")))", imports = ["java.util.Date", "com.android.Color", "custom.package.CustomClass"])
```
and I'm not sure I get how your construction will help here...

Reply:
hm.. yeah. too complicated cases. let's leave as is